cluster/images/OWNERS

```diff
@@ -0,0 +1,7 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
I believe we can exclude all of cluster/images, please verify.

Reply:
Updated to remove /cluster/images